src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/controller/HtmlController.kt

```diff
@@ -73,20 +64,38 @@ class HtmlController(@Autowired val movesForMonthService: MovesForMonthService)
         return "moves"
     }
 
+    @RequestMapping(JOURNEYS_URL)
```

Comment:
For clarity probably worth changing to use the explicit `@GetMapping` and `@PostMapping` annotations in the controllers.